doc/nrf/libraries/networking/azure_iot_hub.rst

```diff
@@ -36,12 +38,19 @@ See `Creating an Azure IoT Hub instance using the Azure portal`_ for more inform
 The connection to Azure IoT Hub with MQTT is secured using TLS.
 For testing purposes, see `Creating Azure IoT Hub certificates`_ for the steps to create certificates and a private key for the leaf device, and to register the generated test root certificate to be used with an IoT hub.
 
-The Azure IoT Hub library requires the provisioning of the following certificates and a private key for a successful TLS connection:
+The Azure IoT Hub library requires the provisioning of the following certificates and private key for a successful TLS connection:
 
-1. `Baltimore CyberTrust Root certificate`_ - Server certificate, used to verify the server's certificate while connecting.
+1. `Baltimore CyberTrust Root Certificate`_ - Server certificate, used to verify the server's certificate while connecting.
 #. Public device certificate - generated by the procedures described in `Creating Azure IoT Hub certificates`_ , used by Azure IoT Hub to authenticate the device.
 #. Private key of the device.
 
+.. important::
+   Azure has started the process of migrating their IoT Hub and DPS server certificates from `Baltimore CyberTrust Root Certificate`_ to `DigiCert Global Root G2`_.
+   Azure advises that all devices should have both Baltimore CyberTrust Root and DigiCert Global Root G2 certificates to avoid disruption of service during the transition.
```

Comment:
```suggestion
   Azure advises to have both Baltimore CyberTrust Root and DigiCert Global Root G2 certificates for all devices to avoid disruption of service during the transition.
```